upgrade/scripts/upgrade/util/sysmgmt-health-upgrade.sh

```diff
@@ -27,34 +27,15 @@
 
 function sysmgmt_health() {
   echo "Checking for chart version of cray-sysmgmt-health"
-  version="45.1"
-  if [ "$(helm ls -o json --namespace sysmgmt-health | jq -r --argjson version $version '.[] | select(.app_version | sub(".[0-9]$";"") | tonumber | . = $version).name')" ]; then
-    prom0_pvc="prometheus-cray-sysmgmt-health-kube-p-prom-db-prometheus-cray-sysmgmt-health-kube-p-prom-0"
-    prom1_pvc="prometheus-cray-sysmgmt-health-kube-p-prom-db-prometheus-cray-sysmgmt-health-kube-p-prom-1"
-    prom0_shard_pvc="prometheus-cray-sysmgmt-health-kube-p-prom-db-prometheus-cray-sysmgmt-health-kube-p-prom-shard-1-0"
-    prom1_shard_pvc="prometheus-cray-sysmgmt-health-kube-p-prom-db-prometheus-cray-sysmgmt-health-kube-p-prom-shard-1-1"
-    alert_pvc="alertmanager-cray-sysmgmt-health-kube-p-alertmanager-db-alertmanager-cray-sysmgmt-health-kube-p-alertmanager-0"
-    thanos_ruler_pvc="thanos-ruler-kube-prometheus-stack-thanos-ruler-data-thanos-ruler-kube-prometheus-stack-thanos-ruler-0"
+  version="0.17"
+  if [ "$(helm ls -o json --namespace sysmgmt-health | jq -r --arg version $version '.[] | select(.app_version | sub(".[0-9]$";"") | tonumber | . = $version).name')" ]; then
```

Comment:
Capture the output of `$( )` in a variable so this conditional is easier to read and debug.

Reply:
done.